### PR TITLE
Add UI of task feature

### PR DESCRIPTION
**Pull request by JoAndNight**
Description: This PR adds UI screens for task feature.

- [x]  Unit tests (For task feature): Before PR: 50% After PR: 60%
- [x]  Exception Handling: Exceptions are handled.
- [x]  JavaDocs Documentation
- [x]  SOLID principles
- [x]  Clean Architecture: Aligned with CA.

### CODE REVIEW

File: src/main/java/use_cases/tasks/delete_task/DeleteTaskOutputBoundary.java [hunk 1]
```diff
@@ -5,9 +5,6 @@
  * This is an interface that passes the output to the Presenter of task deletion.
  */
 public interface DeleteTaskOutputBoundary {
-    // There is no prepareFailView because this use case occurs after the user have selected a specific task. So the
```
Comment (AngelEyesMagicMaker) [inline src/main/java/use_cases/tasks/delete_task/DeleteTaskOutputBoundary.java:8]: Delete comments are really good for code smell.

**Review by AngelEyesMagicMaker — APPROVED** [review]

File: src/main/java/ui/tasks/TaskScreen.java [hunk 2]
```diff
@@ -71,7 +71,6 @@ public void actionPerformed(ActionEvent e) {
             editTaskController.edit(
                     // id
                     name.getText(),
```
Comment (m-banthwan) [inline src/main/java/ui/tasks/TaskScreen.java:73]: Please check that the parameters are not commented out.

**Review by m-banthwan — APPROVED** [review]

File: src/main/java/entities/Task.java [hunk 3]
```diff
@@ -54,13 +50,6 @@ public void markAsIncomplete() {
         this.completeStatus = FALSE;
     }
 
-    /**
-     * Set the deadline of the task to the given deadline.
-     * @param deadline - The deadline that the users wants the task to have.
-     */
-    public void setDeadline(Calendar deadline) {
```
Comment (happynasit) [inline src/main/java/entities/Task.java:61]: Great job at including javadoc in every method

**Review by MadhavKanna — APPROVED** [review]
great javadocs and comments, approved.

File: src/main/java/use_cases/tasks/edit_task/EditTaskInputData.java [hunk 4]
```diff
@@ -10,20 +10,17 @@
 public class EditTaskInputData {
     private final int taskId;
     private final String inputName;
-    private final Calendar inputDeadline;
```
Comment (happynasit) [inline src/main/java/use_cases/tasks/edit_task/EditTaskInputData.java:13]: Good practise in making most of the instance variables private

File: src/main/java/use_cases/tasks/create_task/CreateTaskDsRequestModel.java [hunk 5]
```diff
@@ -13,20 +13,19 @@
 
 public class CreateTaskDsRequestModel {
     private final String name;
-    private final boolean completeStatus = FALSE;
```
Comment (happynasit) [inline src/main/java/use_cases/tasks/create_task/CreateTaskDsRequestModel.java:16]: Great way in using final statements and private instance variables

**Review by happynasit — APPROVED** [review]

**Pull request merged**